### PR TITLE
[button] update icon-only guidance

### PR DESCRIPTION
**Pull request by laurenmrice**
We do not support a danger icon-only button. It only is available in the primary, secondary, tertiary and ghost variants.

**Changed**

- Icon-only buttons: Edited out that we support danger icon-only buttons.

### CODE REVIEW

File: src/pages/components/button/usage.mdx [hunk 1]
```diff
@@ -632,9 +632,7 @@ a text label, but use icon buttons cautiously.
 
 ### Icon-only buttons
 
-Icon buttons allow users to take actions, and make choices, with a single tap.
-Icon buttons can take the form of any of the five variants above but most
-commonly will be styled as primary or ghost buttons.
+Icon buttons allow users to take actions, and make choices, with a single tap. Icon buttons can take the form of a primary, secondary, tertiary or ghost variant but most commonly will be styled as primary or ghost buttons.
```
Comment (janchild) [inline src/pages/components/button/usage.mdx:635]:
```suggestion
Icon buttons allow users to take actions, and make choices, with a single tap. Icon buttons can take the form of a primary, secondary, tertiary, or ghost variant but most commonly will be styled as primary or ghost buttons.
```

**Review by janchild — APPROVED** [review]
@laurenmrice This is great! Just one comma tweak. ;)

**Review by emyarod — APPROVED** [review]
looks good to me after suggested fixes

File: src/pages/components/button/usage.mdx (hunk 1 above)
Comment (laurenmrice) [reply src/pages/components/button/usage.mdx:635]: You got me !!!

**Pull request merged**